api/src/main/java/org/openmrs/module/fhir/api/util/FHIRUtils.java

```diff
@@ -121,6 +121,10 @@ public static String getPractitionerStrategy() {
 		return Context.getAdministrationService().getGlobalProperty("fhir.practitioner.practitionerStrategy");
 	}
 
+	public static String getPlanDefinitionSrategy() {
```

Comment:
There is a typo. Please rename getPlanDefinitionStrategy.

Reply:
Nice, thanks.